studio/components/interfaces/Auth/EmailTemplates/TemplateEditor.tsx

```diff
@@ -45,20 +51,24 @@ const TemplateEditor = ({ template }: TemplateEditorProps) => {
     if (messageProperty) payload[messageSlug] = bodyValue
 
     setSubmitting(true)
-    const { error } = await authConfig.update(payload)
-
-    if (!error) {
-      ui.setNotification({ category: 'success', message: 'Successfully updated settings' })
-      resetForm({
-        values: values,
-        initialValues: values,
-      })
-      setBodyValue(authConfig?.config?.[messageSlug])
-    } else {
-      ui.setNotification({ category: 'error', message: 'Failed to update settings' })
-    }
-
-    setSubmitting(false)
+    updateAuthConfig(
+      { projectRef: projectRef!, config: payload },
+      {
+        onError: () => {
```

Comment:
likewise as per https://github.com/supabase/supabase/pull/16227/files#r1289813556
shift all these to the mutation declaration

use isLoading from the mutation rather than isSubmitting from the Form

Reply:
Can't, due to `resetForm`.  I've removed `isSubmitting`.